crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -521,6 +521,7 @@ def valid_transition_states
       "applying", "discovered", "discovering", "hardware-installed",
       "hardware-installing", "hardware-updated", "hardware-updating",
       "installed", "installing", "ready", "readying", "recovering",
+      "crowbar_upgrade",
```

Comment:
I think the node state should be consistent with js we use in UI, so we don't need to add any translation functions...